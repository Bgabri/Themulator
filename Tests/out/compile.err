==1366129== Memcheck, a memory error detector
==1366129== Copyright (C) 2002-2024, and GNU GPL'd, by Julian Seward et al.
==1366129== Using Valgrind-3.23.0 and LibVEX; rerun with -h for copyright info
==1366129== Command: ./thml.out -D program --compiler gcc
==1366129== 
==1366129== 
==1366129== HEAP SUMMARY:
==1366129==     in use at exit: 0 bytes in 0 blocks
==1366129==   total heap usage: 18 allocs, 18 frees, 41,230 bytes allocated
==1366129== 
==1366129== All heap blocks were freed -- no leaks are possible
==1366129== 
==1366129== For lists of detected and suppressed errors, rerun with: -s
==1366129== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
