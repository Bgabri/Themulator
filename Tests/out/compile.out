[32m[1m[o][0m 1.in
[32m[1m[o][0m 2.in
[32m[1m[o][0m 3.in
[35m[1m[e][0m exitError.in	[2mRuntime error: Program exited with code: 1[0m
[33m[1m[?][0m noOut.in	[2mReference file not found.[0m
[31m[1m[x][0m wrongOut.in	[2mWrong output[0m
